todo_app/app/controllers/tasks_controller.rb

```diff
@@ -2,12 +2,12 @@
 
 class TasksController < ApplicationController
   def index
-    @tasks = Task.search(nil, nil, create_sort_query)
+    @tasks = Task.page(params[:page]).search(nil, nil, create_sort_query)
```

Comment:
もしかしたら結果にはあまり関係ないのかもしれないですが、以下のように最後に付けた方がいいかも？
処理の流れとしても、検索結果に対してページ毎に分割しているとした方が自然っぽい。
`search` の前に書いた理由って何かあったりしましたー？
```suggestion
    @tasks = Task.search(nil, nil, create_sort_query).page(params[:page])
```

Reply:
> 処理の流れとしても、検索結果に対してページ毎に分割しているとした方が自然っぽい。
たしかにそうですね :thinking_face: 特に理由はなかったので修正しておきます〜〜